apps/web/src/components/workflow/preview/ErrorPrettyRender.tsx

```diff
@@ -74,7 +75,8 @@ export function ErrorPrettyRender({ error: unparsedError }) {
                 fontFamily: 'mono',
               })}
             >
-              {JSON.stringify(error.data, null, 2)}
+              {error.data?.stack}
+              {!error.data?.stack && JSON.stringify(error.data, null, 2)}
```

Comment:
A little bit yucky here, but we can tidy this up when we rebuild Local Studio to use proper type-guards.